packages/@jsii/go-runtime/jsii-runtime-go/go.mod

```diff
@@ -6,3 +6,5 @@ require (
 	github.com/Masterminds/semver/v3 v3.1.1
 	github.com/stretchr/testify v1.7.0
 )
+
+retract v1.27.0
```

Comment:
How the hell did you know this is a thing...

Reply:
I tried to run CDK synth as per the preview instructions and noticed signature mismatch errors when installing this dependency.